modules/misc/news.nix

```diff
@@ -1135,6 +1135,14 @@ in
           A new modules is available: 'programs.zsh.antidote'
         '';
       }
+
+      {
+        time = "2023-06-30T14:46:22+00:00";
+        condition = config.programs.zsh.enable;
+        message = ''
+          A new modules is available: 'programs.zsh.antidote'
```

Comment:
```suggestion
          A new module is available: 'services.ssh-agent'
```

Reply:
Go figure that I'd forget to adjust the actual content of the news item :upside_down_face: thanks